daemon/mgr/container_types.go

```diff
@@ -166,6 +166,17 @@ type ContainerStatsConfig struct {
 	OutStream io.Writer
 }
 
+// ContainerPathStat is used to encode the header from
+// GET "/containers/{name:.*}/archive"
+// "Name" is the file or directory name.
+type ContainerPathStat struct {
+	Name  string      `json:"name"`
+	Path  string      `json:"path"`
+	Size  int64       `json:"size"`
+	Mode  os.FileMode `json:"mode"`
+	Mtime time.Time   `json:"mtime"`
```

Comment:
I wish that more annotation of the field could be added into the code line. Thanks